environment.linux.lock.yml

```diff
@@ -3,89 +3,164 @@ channels:
   - bioconda
   - defaults
 dependencies:
-  - _libgcc_mutex=0.1=main
+  - _libgcc_mutex=0.1=conda_forge
+  - _openmp_mutex=4.5=0_gnu
+  - alsa-lib=1.1.5=h516909a_1002
   - appdirs=1.4.3=py_1
   - attrs=19.3.0=py_0
   - blas=2.14=openblas
   - bowtie2=2.3.5=py36he860b03_0
-  - bwa=0.7.17=hed695b0_6
-  - bzip2=1.0.8=h516909a_1
+  - bwa=0.7.17=hed695b0_7
+  - bzip2=1.0.8=h516909a_2
   - ca-certificates=2019.11.28=hecc5488_0
   - certifi=2019.11.28=py36_0
   - cffi=1.13.2=py36h8022711_0
   - chardet=3.0.4=py36_1003
+  - click=7.0=py_0
+  - coloredlogs=10.0=py36_1000
+  - colormath=3.0.0=py_2
   - configargparse=0.13.0=py_1
-  - cryptography=2.8=py36h72c5cf5_0
+  - cryptography=2.8=py36h72c5cf5_1
   - curl=7.65.3=hf8cf82a_0
   - cutadapt=2.5=py36h516909a_0
+  - cycler=0.10.0=py_2
   - datrie=0.8=py36h516909a_0
+  - dbus=1.13.6=he372182_0
+  - decorator=4.4.1=py_0
   - dnaio=0.4.1=py36h516909a_0
-  - docutils=0.15.2=py36_0
+  - docutils=0.16=py36_0
   - ema=0.6.2=h8b12597_1
+  - expat=2.2.5=he1b5a44_1004
+  - fastqc=0.11.8=2
+  - font-ttf-dejavu-sans-mono=2.37=hab24e00_0
+  - fontconfig=2.13.1=h86ecdb6_1001
   - freebayes=1.3.1=py36h56106d0_0
+  - freetype=2.10.0=he983fc9_1
+  - future=0.18.2=py36_0
+  - gettext=0.19.8.1=hc5be6a0_1002
+  - giflib=5.2.1=h516909a_1
   - gitdb2=2.0.6=py_0
   - gitpython=3.0.5=py_0
-  - hapcut2=1.2=hed50d52_0
-  - htslib=1.9=h244ad75_9
+  - glib=2.58.3=py36h6f030ca_1002
+  - gst-plugins-base=1.14.5=h0935bb2_0
+  - gstreamer=1.14.5=h36ae1b5_0
+  - hapcut2=1.1=hed695b0_0
```

Comment:
Why is this downgraded?

Reply:
For some reason it downgraded when I added multiqc and fastqc, not quite sure why. I'll see if running the condalock upgrades it again.

Reply:
So seemingly conda downgrades hapcut2 from 1.2 to 1.1 when these programs are in the environment files, and I don't know why. Possibly it could have to do with

I solved it by specifying version to 1.2 in the main environment file, which seems to work.

Reply:
Strange. Good that specifying to 1.2 worked.